tests/testthat/_snaps/trace.md

```diff
@@ -48,19 +48,20 @@
     Output
            x
         1. \-rlang:::f()
-        2.   \-rlang:::g() test-trace.R:49:20
-        3.     +-[ base::tryCatch(...) ] with 6 more calls test-trace.R:50:20
+        2.   \-rlang:::g() at test-trace.R:49:20
+        3.     +-[ base::tryCatch(...) ] with 6 more calls at test-trace.R:50:20
        10.     \-rlang:::h()
-       11.       +-[ base::tryCatch(...) ] with 3 more calls test-trace.R:51:20
+       11.       +-[ base::tryCatch(...) ] with 3 more calls at test-trace.R:51:20
        15.       \-rlang:::i()
-       16.         +-[ base::tryCatch(...) ] with 1 more call test-trace.R:52:20
+       16.         +-[ base::tryCatch(...) ] with 1 more call at test-trace.R:52:20
        18.         \-rlang::trace_back(e, bottom = 0)
     Code
       # Branch
       print(trace, simplify = "branch", dir = dir, srcrefs = srcrefs)
     Output
         1. rlang:::f()
-        2. rlang:::g() test-trace.R:49:20
+        2. rlang:::g()
```

Comment:
Why is this spread over two lines?

Reply:
Because calls and srcrefs tend to be long, this way it's easier to read. The srcref line is also dimmed (styled in grey).

Reply:
Ok. I just asked because it looks different to the full trace above.